HW_4/task_3.py

```diff
@@ -0,0 +1,51 @@
+"""
+Задать натуральное число k.
+Сформируйте многочлен (полином) степени k со случайными коэффициентами из промежутка от 0 до 100, включительно.
+Многочлен вывести в консоль и записать в файл.
+
+Ввод: значение типа <int>
+Вывод: значение типа <str>, файл с одной строкой.
+
+Пример:
+2
+2x^2 + 4x + 5 = 0
+"""
+from random import randint
+import itertools
+
+k = randint(2, 7)
+
+def get_ratios(k):
+    ratios = [randint(0, 101) for i in range (k + 1)]
+    while ratios[0] == 0:
+        ratios[0] = randint(1, 10) 
```

Comment:
```suggestion
    if ratios[0] == 0:
        ratios[0] = randint(1, 10) 
```
Тут цикл ни к чему. Все равно будет только одна итерация или не будет вообще.

Reply:
Но я не согласен с тем, что вы почему-то убираете 0, если он является первым коэффициентом.